config.toml

```diff
@@ -1,11 +1,13 @@
-baseURL = "https://hwayne.github.io/alloy-hugo-demo/"
+baseURL = "https://hwayne.github.io/alloy-hugo-demo"
```

Comment:
For the record, once we migrate to the actual site we'll have to change this, uh, the actual site

Reply:
sorry to screw that up. was playing with paths to try and get the image shortcode to work.